e2e_tests/tests/createAndRemove.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { test, expect } from "@playwright/test";
+
+test.setTimeout(5000);
+
+const createAndRemove = [
+	{ type: "sidebar", locator: `div.CoreSidebar.component` },
+	{ type: "section", locator: `section.CoreSection.component` },
+	{ type: "columns", locator: `div.CoreColumns.component` },
+];
+
+const fullCheck = [
+	{ type: "button", locator: `button.CoreButton.component` },
+	{ type: "text", locator: `div.CoreText.component` },
+	{ type: "header", locator: `div.CoreHeader.component` },
+	{ type: "heading", locator: `div.CoreHeading.component` },
+	{ type: "dataframe", locator: `div.CoreDataframe.component` },
+	{ type: "html", locator: `div.CoreHTML.component` },
+	{ type: "pagination", locator: `div.pagination.component` },
+	{ type: "repeater", locator: `div.CoreRepeater.component` },
+	//{ type: "column", locator: `div.CoreColumn.component` },
```

Comment:
If you want to automate something really cool you could check the `allowedParentTypes` and automatically create the parents.

e.g. from `tab`:

`allowedParentTypes: ["tabs", "repeater"]`

So you could exclude `repeater` and you have the right parent.

That'd work for `column`, `step`, `tab`.

Reply:
This would be really nice but lets save it as future improvement, first need to stabilize what we already have.